swagger.yaml

```diff
@@ -385,6 +385,54 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /v1/users/{userId}/readonly-access:
+    put:
+      summary: Update user's read-only access status
+      parameters:
+        - name: userId
+          in: path
+          description: ID of the user to update
+          required: true
+          type: integer
+        - name: body
+          in: body
+          description: Request body containing the new read-only access status
+          required: true
+          schema:
+            type: object
+            properties:
+              is_readonly:
+                type: boolean
+                description: New read-only access status for the user
+      tags:
+        - Users
+      security:
+        - Bearer: [ ]
+      operationId: UpdateReadonlyAccess
+      responses:
+        204:
+          description: Success
+        400:
+          description: Bad request
+          schema:
+            $ref: "#/definitions/Error"
+        401:
```

Comment:
403 error will be added automatically right after the server starts. You do not have to add it manually.